From 90bded7cf8845c232a1a19b6b225fbbc08938a3a Mon Sep 17 00:00:00 2001
From: "jeffbai@aosc.io" <jeffbai@aosc.io>
Date: Thu, 24 Oct 2024 21:07:18 +0800
Subject: [PATCH 062/118] FROMLIST: drm: amdgpu: fix comment about
 amdgpu.abmlevel defaults

Since 040fdcde288a2830edc31dd507963d6aadf990d2 ("drm/amdgpu: respect the
abmlevel module parameter value if it is set"), the default value for
amdgpu.abmlevel was set to -1, or auto. However, the comment explaining
the default value was not updated to reflect the change (-1, or auto; not
-1, or disabled).

Clarify that the default value (-1) means auto.

Fixes: 040fdcde288a2830edc31dd507963d6aadf990d2 ("drm/amdgpu: respect the abmlevel module parameter value if it is set")
Reported-by: Ruikai Liu <rickliu2000@outlook.com>
Signed-off-by: Mingcong Bai <jeffbai@aosc.io>

Link: https://lore.kernel.org/amd-gfx/43965fdb4f4a7c4223885c8e417bdc69@aosc.io/
Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
---
 drivers/gpu/drm/amd/amdgpu/amdgpu_drv.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/amdgpu/amdgpu_drv.c b/drivers/gpu/drm/amd/amdgpu/amdgpu_drv.c
index 81d9877c8735..b248d7096abc 100644
--- a/drivers/gpu/drm/amd/amdgpu/amdgpu_drv.c
+++ b/drivers/gpu/drm/amd/amdgpu/amdgpu_drv.c
@@ -892,7 +892,7 @@ module_param_named(visualconfirm, amdgpu_dc_visual_confirm, uint, 0444);
  * the ABM algorithm, with 1 being the least reduction and 4 being the most
  * reduction.
  *
- * Defaults to -1, or disabled. Userspace can only override this level after
+ * Defaults to -1, or auto. Userspace can only override this level after
  * boot if it's set to auto.
  */
 int amdgpu_dm_abm_level = -1;
-- 
2.47.0

